pkg/sdk/channels_test.go

```diff
@@ -398,7 +398,7 @@ func TestListChannels(t *testing.T) {
 			channelsPageMeta: channels.Page{
 				Actions: []string{},
 				Order:   "updated_at",
-				Dir:     "asc",
+				Dir:     "desc",
```

Comment:
We should add test for this, i.e. test the order of returned elements, but we should do it in a separate PR.